lib/shared_state.js

```diff
@@ -36,26 +36,41 @@ export class SharedState {
     // Owner of the state: either a client socket id or 'server'.
     this.owner_ = owner;
   }
+  earliest() {
+    return this.store_[0];
```

Comment:
why no guard as with latest()?

Reply:
Because it will return undefined if it's not set anyway.